alcs-frontend/src/styles.scss

```diff
@@ -1,7 +1,6 @@
 // Custom Theming for Angular Material
 // For more information: https://material.angular.io/guide/theming
 @use '@angular/material' as mat;
-@use '@matheo/datepicker/theming' as datepicker;
```

Comment:
Why do you need to remove this one? I remember from the documentation that theming is required, and it can be adjusted as any angular material

Reply:
I was getting duplicate style errors, in theory the mtx / material extensions library says to remove this when you add that styling as it should include both, however I noticed without this the inline date picker has some issues. Reverting this as I'd rather have duplicate style warnings than a broken date picker.